spec/api/v1/external_users/claims/integration/advocate_claim_creation_spec.rb

```diff
@@ -462,7 +462,6 @@ def results(claim, expected)
 
     context 'final fee claims' do
       include_context 'deactivate deprecation warnings'
-      include_examples 'scheme 9 advocate final claim', relative_endpoint: :advocate
```

Comment:
the shared example was only there because of the duplicated endpoints. So they are not "shared" anymore.  To be consistent with the other specs in this file we can therefore get rid of all the shared examples:
```
'scheme 9 advocate final claim'
'scheme 10 advocate final claim'
'scheme 12 advocate final claim'
```
and put there test/example directly into the  relevant content, example:

```
context 'scheme 9' do
  let(:representation_order_date) { Date.new(2018, 03, 31).beginning_of_day }
  let(:advocate_category) { 'Junior alone' }
  let(:offence) { create(:offence, :with_fee_scheme_nine) }

  # context 'final fee claims' do  DONT NEED THIS CONTEXT EITHER FROM WHAT I CAN TELL
  specify 'Case management system creates a valid scheme 9 graduated fee claim' do
    post ClaimApiEndpoints.for('advocates/final').create, claim_params.merge(offence_id: offence.id), format: :json
    ...
```